packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -223,7 +223,7 @@ function preprocess<T extends z.ZodType>({
             if (optionValue) {
               // `typeOfOptionInput` can be any of the main types. So, we the same validations should run for `optionValue`
               if (typeOfOptionInput === "phone") {
-                if (!phoneSchema.safeParse(optionValue).success) {
+                if (!(await phoneSchema.safeParseAsync(optionValue)).success) {
```

Comment:
There was one occurence earlier in the code for phoneSchema that has been handled. That triggered when a Phone number type field is chosen but this one is special location field handling.